src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -7,7 +7,7 @@ import Iframe from "react-iframe";
 interface PharmacyProps {
   // Id used for creating React keys
   // eslint-disable-next-line react/no-unused-prop-types
-  id: string;
+  id: string | number;
```

Comment:
Why do we need the ID here? I don't see it being used. The key prop that is set in the `PharmacyList` should be sufficient to setting the react key.

Reply:
Well, the key prop is set in `PharmacyList` like this:

```
pharmacyList.map((pharmacy) => {
            return (
              <PharmacyCard
                  key={pharmacy.id}
```

Address could be used instead? Dunno if phone or website would be appropriate?

Reply:
Ah I see now.. since `pharmacyList` is depending on the props from `PharmacyCard`. I guess this is ok. My point was more about this interface having a prop that would be never used in the component but was there because a consumer depended on it.

The ID coming from the server is probably the most appropriate key here still I'd imagine.